index.js

```diff
@@ -381,5 +381,32 @@ form.addEventListener('submit', (event) => {
     error.textContent = '';
     error.className = 'error';
     form.submit();
+    localStorage.removeItem('data');
   }
-});
\ No newline at end of file
+});
+
+// Preserve data in web browser
+
+// Adding data to local storage
+form.addEventListener('keyup', () => {
+  const data = {
+    name: document.getElementById('full-name').value,
+    email: document.getElementById('email').value,
+    message: document.getElementById('message').value,
+  };
+  localStorage.setItem('data', JSON.stringify(data));
+});
+
+function fillForm() {
+  const data = JSON.parse(localStorage.getItem('data'));
+  if (data) {
+    const name = form.elements['full name'];
+    const { email } = form.elements;
+    const { message } = form.elements;
+    name.value = data.name;
+    email.value = data.email;
+    message.value = data.message;
+  }
+}
```

Comment:
- [ ] As required for the project, kindly make sure the **contact form information is persistent to page reloads**, **When visiting the page the form inputs should be contained, auto-populated with, the values entered in the previous visit to the page.**
Right now, when a user submitted a form, it immediately deletes all data saved in the Localstorage instead of remaining saved in your browser. Make sure you fix that.